### PR TITLE
Export and import select fields

### PR DESCRIPTION
**Pull request by nestordedios**

### CODE REVIEW

File: src/Export.php [hunk 1]
```diff
@@ -94,10 +94,14 @@ private function buildContent(?string $contentType)
             /** @var Content $record */
             foreach ($contentEntities as $record) {
                 $currentITem = $record->toArray();
+                // Get the select fields that have an entity referenced
+                $selectFields = $this->getSelectFields($record);
+                // Update the $currentItem with the right data that will be needed to make the reference when importing
+                $currentITem = $this->updateSelectFields($currentITem, $selectFields);
```
Comment (I-Valchev) [inline src/Export.php:100]: Looks good! 

Should it be `currentItem` everywhere instead of `currentITem` @nestordedios ?

Comment (nestordedios) [reply src/Export.php:100]: I think it should be `currentItem` everywhere, I used `currentITem` as it was defined on line 96.

I will make the modifications and push a new commit.

Comment (nestordedios) [reply src/Export.php]: @I-Valchev , I have renamed the variable and pushed a new commit

Comment (I-Valchev) [reply src/Export.php]: Nice, looks good! @bobdenotter can you review and merge?

**Review by bobdenotter — APPROVED** [review]
Looks good. Let's merge in, and continue testing! 👍

**Comment by bobdenotter** [comment]
Tagged a new release: https://github.com/bobdenotter/conimex/releases/tag/2.0.0-beta.3

**Pull request merged**